operators/src/source/ogr_source.rs

```diff
@@ -3772,7 +3867,7 @@ mod tests {
 
         let query_bbox = BoundingBox2D::new((-180.0, -90.0).into(), (180.00, 90.0).into()).unwrap();
 
-        let context = MockQueryContext::new((1024 * 1024).into());
+        let context = MockQueryContext::new(ChunkByteSize::default());
```

Comment:
I wouldn't replace this and the other's with the call to `::default`. I think it was a bad idea to introduce this default implementation for `ChunkByteSize` as it makes no sense. We should keep it as fixed values in order to make it easier to remove it in the future.

https://trello.com/c/zlKPcK44/483-default-von-chunkbytesize-entfernen-und-mock-konstante-einf%C3%BCgen